api/config/genesis.go

```diff
@@ -3,16 +3,15 @@ package config
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/spacemeshos/go-spacemesh/log"
-	"math"
-	"math/big"
 	"os"
+
+	"github.com/spacemeshos/go-spacemesh/log"
 )
 
 // GenesisAccount is the json representation of an account
 type GenesisAccount struct {
-	Balance *big.Int `json:"balance" gencodec:"required"`
-	Nonce   uint64   `json:"nonce"`
+	Balance uint64 `json:"balance" gencodec:"required"`
```

Comment:
Is this still `required` now that it's no longer a pointer?

Reply:
As mentioned, we should ask Noam to double check? Leaving this in for now.

Reply:
This appears to be Anton's code from a while ago (#491). It doesn't appear to be used anywhere else in the codebase or mentioned anywhere in the docs so I think it's safe to remove the entire `gencodec` statement. @antonlerner let us know if you disagree/if we've missed something here!

Reply:
For reference: https://godoc.org/github.com/fjl/gencodec